src/cmds/qstat.c

```diff
@@ -804,7 +806,31 @@ altdsp_statjob(struct batch_status *pstat, struct batch_status *prtheader, int a
 		pstat = bs_isort(pstat, cmp_est_time);
 
 	if (prtheader) {
-		printf("\n%s: ", prtheader->name);
+		svr_conn_t *svr_connections = get_conn_servers(conn);
+		int num_cfg_svrs = get_num_servers();
```

Comment:
same comment as above. Could we do away with get_num_servers() here and just return a null terminated array? Of course, this is no problem per se either